go/vt/sqlparser/parse_test.go

```diff
@@ -819,20 +819,20 @@ var (
 	}, {
 		input: "set sql_safe_updates = 1",
 	}, {
-		input:  "alter ignore table a add foo",
-		output: "alter table a",
+		input:  "alter ignore table a add foo int",
```

Comment:
I changed some of these tests because AFAICT they aren't valid SQL. In particular, I added some types in the ADDs and removed some types in the DROPs.

Reply:
Yeah the parser originally ignored large parts of alter table statements. I think I was too lazy to fix these tests when I added alter table support :p